### PR TITLE
Setup environment of the project

### PR DESCRIPTION
**Pull request by pryaniq**

### CODE REVIEW

**Comment by pryaniq** [comment]
[Setup environment of the project](https://trello.com/c/0HrA3gff/6-setup-environment-of-the-project)

File: .swiftlint.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+type_name:
+  max_length:
+    warning: 60
+    error: 80
+identifier_name:
+  min_length:
+    warning: 1
+
+nesting:
```
Comment (DaianAiziatov) [inline .swiftlint.yml:64]: is it mean that 2 nested for-in loop make warning?

File: .gitignore [hunk 2]
```diff
@@ -32,37 +28,8 @@ xcuserdata/
...
-Carthage/Build
-
-# fastlane
-#
-# It is recommended to not store the screenshots in the git repo. Instead, use fastlane to re-generate the
-# screenshots whenever they are needed.
-# For more information about the recommended setup visit:
-# https://docs.fastlane.tools/best-practices/source-control/#source-control
+## CocoaPods
+Pods/
```
Comment (DaianAiziatov) [inline .gitignore:32]: why we ignore pods files? or I misunderstood this line

**Review by DaianAiziatov — APPROVED** [review]
approved

**Pull request merged**